### ISSUE TITLE
Code on examples website does not work

### ISSUE THREAD

**Issue by toefel18** [tannerlinsley/react-charts#64]
The sample code on https://react-charts.js.org/ does not work.

I tried the AreaChart and found:
1. hooks/useChartConfig cannot be imported
1. The dependency Box cannot be found 
1. I also see a weird option: sourcecode, used for syntax highlighting (this seems like the code contains extra set-up for the website itself.

I know the hook it is this complex beast: 
https://github.com/tannerlinsley/react-charts/blob/master/www/src/hooks/useChartConfig.js

Is it possible to add the simplest one specific for this graph?

**Comment by SiddharthMantri** [comment]
Sample code for AreaChart with 2 series. 
[CodeSandbox link](https://codesandbox.io/s/charming-leakey-ee4h2)

`
import React from "react";
import "./styles.css";
import { Chart } from "react-charts";

export default function App() {
  const series = React.useMemo(
    () => ({
      type: "area"
    }),
    []
  );
  const axes = React.useMemo(
    () => [
      { primary: true, position: "bottom", type: "time" },
      { position: "left", type: "linear", stacked: true }
    ],
    []
  );
  const data = React.useMemo(
    () => [
      {
        label: "Series 1",
        datums: [
          {
            x: new Date("2020-03-18T11:00:00.000Z"),
            y: 60
          },
          {
            x: new Date("2020-03-18T11:30:00.000Z"),
            y: 23
          },
          {
            x: new Date("2020-03-18T12:00:00.000Z"),
            y: 65
          },
          {
            x: new Date("2020-03-18T12:30:00.000Z"),
            y: 84
          },
          {
            x: new Date("2020-03-18T13:00:00.000Z"),
            y: 87
          },
          {
            x: new Date("2020-03-18T13:30:00.000Z"),
            y: 84
          },
          {
            x: new Date("2020-03-18T14:00:00.000Z"),
            y: 96
          },
          {
            x: new Date("2020-03-18T14:30:00.000Z"),
            y: 88
          },
          {
            x: new Date("2020-03-18T15:00:00.000Z"),
            y: 63
          },
          {
            x: new Date("2020-03-18T15:30:00.000Z"),
            y: 60
          }
        ]
      },
      {
        label: "Series 2",
        datums: [
          {
            x: new Date("2020-03-18T11:00:00.000Z"),
            y: 41
          },
          {
            x: new Date("2020-03-18T11:30:00.000Z"),
            y: 15
          },
          {
            x: new Date("2020-03-18T12:00:00.000Z"),
            y: 95
          },
          {
            x: new Date("2020-03-18T12:30:00.000Z"),
            y: 96
          },
          {
            x: new Date("2020-03-18T13:00:00.000Z"),
            y: 33
          },
          {
            x: new Date("2020-03-18T13:30:00.000Z"),
            y: 96
          },
          {
            x: new Date("2020-03-18T14:00:00.000Z"),
            y: 32
          },
          {
            x: new Date("2020-03-18T14:30:00.000Z"),
            y: 49
          },
          {
            x: new Date("2020-03-18T15:00:00.000Z"),
            y: 18
          },
          {
            x: new Date("2020-03-18T15:30:00.000Z"),
            y: 69
          }
        ]
      }
    ],
    []
  );
  return (
    <div className="App">
      <div style={{ width: "500px", height: "500px" }}>
        <Chart data={data} series={series} axes={axes} tooltip />
      </div>
    </div>
  );
}

`

**Comment by dzoba** [comment]
Adding that I also encountered this / was held up by it.

Thanks for the great work on this library!

**Comment by wfendler** [comment]
The use of a custom hooks file that isn't shown really hides a lot of what would be useful to see. It also looks like that codesandbox link is no longer working as well.

**Comment by tannerlinsley** [comment]
Due to a massive overhaul and rewrite of React Charts to TypeScript and a massively upgraded public API, this issue is being bulk closed for triage. The latest version can be tested via the `react-charts@beta` tag. Documentation is in the process of being upgraded, but thankfully, everything is fully typed now, so you can explore the API that way for now.

If, *after testing or playing with the v3 beta*, you believe your issue is still not addressed or relevant, feel free to open a new issue.
